BlocSpot/app/src/main/java/com/izor066/android/blocspot/api/DataSource.java

```diff
@@ -0,0 +1,52 @@
+package com.izor066.android.blocspot.api;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+
+import com.izor066.android.blocspot.BuildConfig;
+import com.izor066.android.blocspot.api.model.PointsOfInterest;
+import com.izor066.android.blocspot.api.model.database.DatabaseOpenHelper;
+import com.izor066.android.blocspot.api.model.database.table.PointsOfInterestTable;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public class DataSource {
+
+    private DatabaseOpenHelper databaseOpenHelper;
+    private PointsOfInterestTable pointsOfInterestTable;
+
+
+
+    public DataSource (Context context) {
+        pointsOfInterestTable = new PointsOfInterestTable();
+        databaseOpenHelper = new DatabaseOpenHelper(context, pointsOfInterestTable);
+
+        if (BuildConfig.DEBUG && true) {
+            context.deleteDatabase("blocspot_db");
```

Comment:
It is a good practise to extract this as a constant in order to have only one point of edit (for example if you decide to change the name of your database, you should only change it once in one place).
I see now that actually it is extracted [here](https://github.com/i-zor066/BlocSpot/pull/1/files#diff-7b610108acc9fc52c7141e4a1a7a16f5R14). As a first step you could declare that variable `public` and reference it here as `context.deleteDatabase(DatabaseOpenHelper.NAME)`. This would eliminate the possibility to change the name in one place, but to forget to change it in the other one.